lonestar/analytics/cpu/cdlp/CMakeLists.txt

```diff
@@ -2,3 +2,5 @@ add_executable(cdlp-cpu cdlp_cli.cpp)
 add_dependencies(apps cdlp-cpu)
 target_link_libraries(cdlp-cpu PRIVATE Katana::galois lonestar)
 
+add_test_scale(small cdlp-cpu NO_VERIFY INPUT rmat15 INPUT_URI "${RDG_RMAT15}" --algo=Synchronous)
```

Comment:
All all the algorithm for testing here. Is there Asynhronous as well?

Reply:
Asynchronous has not been implemented yet.

Reply:
the tests for asynchronous will be added once the asynchronous gets implemented.